shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -215,30 +217,59 @@ private void setSystemChromeApplicationSwitcherDescription(
     }
   }
 
-  private void setSystemChromeChangeListener() {
-    // Set up a listener to notify the framework when the system ui has changed.
+  private void setSystemChromeChangeListenerLegacy() {
     View decorView = activity.getWindow().getDecorView();
     decorView.setOnSystemUiVisibilityChangeListener(
         new View.OnSystemUiVisibilityChangeListener() {
           @Override
           public void onSystemUiVisibilityChange(int visibility) {
             if ((visibility & View.SYSTEM_UI_FLAG_FULLSCREEN) == 0) {
-              // The system bars are visible. Make any desired adjustments to
-              // your UI, such as showing the action bar or other navigational
-              // controls. Another common action is to set a timer to dismiss
-              // the system bars and restore the fullscreen mode that was
-              // previously enabled.
               platformChannel.systemChromeChanged(false);
             } else {
-              // The system bars are NOT visible. Make any desired adjustments
-              // to your UI, such as hiding the action bar or other
-              // navigational controls.
               platformChannel.systemChromeChanged(true);
             }
           }
         });
   }
 
+  private void setSystemChromeChangeListener() {
```

Comment:
Shouldn't the test call this method to activate the listener?

Reply:
The test calls the method that calls this method!